apps/core/mutations.py

```diff
@@ -143,10 +148,59 @@ def mutate(instance, root, info, id, name):
         return DeleteTableFromWorkspace(result=instance, errors=None, ok=True)
 
 
+ActionEnum = graphene.Enum("ActionEnum", [(ac, ac) for ac in get_all_action_names()])
+
+
+class ActionInputType(graphene.InputObjectType):
+    action_name = graphene.Field(ActionEnum)
+    params = graphene.List(graphene.String)
+
+
+class PerformTableAction(graphene.Mutation):
```

Comment:
@bewakes How about `ApplyTableAction`?

Reply:
Well, I thought about this, but applying makes sense when we create a snapshot right away which we don't do. Thus kept PerformTableAction.